frontend/relay-workflows-lib/lib/components/WorkflowRelay.tsx

```diff
@@ -0,0 +1,88 @@
+import { useFragment } from "react-relay";
+import { graphql } from "react-relay";
+import { WorkflowAccordion } from "workflows-lib";
+import type { WorkflowStatus } from "workflows-lib";
+import { WorkflowRelayFragment$key } from "./__generated__/WorkflowRelayFragment.graphql";
+
+const WorkflowRelay = (props: {
+  workflow: WorkflowRelayFragment$key;
+  children: React.ReactNode;
+}) => {
+  const workflowFragment = graphql`
```

Comment:
Can we remove a lot of the fields from this query? Doesn't seem like any of the `status { ... on }` clauses are required here